src/main/java/com/mabl/integration/jenkins/MablStepBuilder.java

```diff
@@ -605,4 +651,14 @@ private static MablRestApiClient createMablRestApiClient(
                 disableSslVerification);
     }
 
+    private void updateBrowser(String browserName , boolean selected){
+        if(selected){
+            if(browsers == null) browsers = new ArrayList<>();
```

Comment:
As a matter of code style, we always use `{}`
```suggestion
            if(browsers == null) { browsers = new ArrayList<>(); }
```

Reply:
Sure, fixed it.